rskj-core/src/main/java/co/rsk/core/bc/FamilyUtils.java

```diff
@@ -150,6 +150,8 @@
             List<Block> uncles = store.getChainBlocksByNumber(ancestor.getNumber());
 
             for (Block uncle : uncles) {
+                if (uncle == null)
```

Comment:
When does this happen?

Reply:
It shouldnt happen, line will be deleted :+1: